src/classify.jl

```diff
@@ -16,3 +16,44 @@ function classify!(array, weights, mask = nothing)
     end
     array
 end
+
+"""
+    blend(arrays[, scaling])
+
+Blend arrays weighted by scaling factors.
+"""
+function blend(arrays, scaling = ones(length(arrays)))
```

Comment:
test for `length(arrays) == length(scaling)` sounds like a good idea here?

Reply:
I see the point here, but is that error message really that more helpful?
```julia
julia> scaling = 1:4;
julia> arrays = [rand(3) for i in 1:3];

julia> @assert length(arrays) == length(scaling)
ERROR: AssertionError: length(arrays) == length(scaling)
Stacktrace:
 [1] top-level scope
   @ REPL[4]:1

julia> arrays .* scaling
ERROR: DimensionMismatch("arrays could not be broadcast to a common size; got a dimension with lengths 3 and 4")
Stacktrace:
```

Reply:
I was thinking more along the lines of 

~~~ julia
if length(scaling) != length(arrays)
    throw(DimensionMismatch("The array of landscapes (n = $(length(arrays))) and scaling (n = $(length(scaling)) must have the same length")
end
~~~

which is more informative than both.